docs/monthly-meeting/2024-01.md

```diff
@@ -0,0 +1,92 @@
+- [**Discourse thread**](https://discuss.python.org/t/42191) (for January)
+- [**Meeting reports**](https://docs-community.readthedocs.io/en/latest/monthly-meeting/) (the latest one might be an [**unmerged PR**](https://github.com/python/docs-community/pulls))
+- **Calendar event:** (send your e-mail to Mariatta for an invitation)
+- **How to participate:**
+  -  Go to [Google Meet](https://meet.google.com/dii-qrzf-wkw) and ask to be let in.
+  -  To edit notes, click the “pencil” or “split view” button on the [HackMD document](https://hackmd.io/@encukou/pydocswg1). You need to log in (e.g. with a GitHub account).
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+
+## Roll call
+
+(Name / `@GitHubUsername` *[/ Discord, if different]*)
+- Carol Willing / `@willingc` / carolwilling
+- Petr Viktorin / `@encukou`
+- Ezio Melotti / `@ezio-melotti`
+- Hugo van Kemenade / `@hugovk`
+- Guido van Rossum / `@gvanrossum`
+- Daniele / `@EvilDMP` / danieleprocida
+- Ege Akman / `@egeakman`
+- Ned / `@nedbat`
+- Ryan / `@ryan-duve`
+- Jim / `@JDLH`
+- CAM / `@CAM-Gerlach`
+- Usman Akinyemi / `@Unique-Usman`
+- Bradley / `@shenanigansd`
+
+## Reports and celebrations
+
+- The Steering Council has [accepted](https://discuss.python.org/t/pep-732-the-python-documentation-editorial-board/36710/9?u=hugovk) [PEP 732](https://peps.python.org/pep-0732/) ("The Python Documentation Editorial Board") 🎉
+  - [Carol] First meeting will be January 8, 2024.
+  - [Carol] Looking at docs and website from a new user's perspective
+  - [Hugo] What would be the best way to contact the Board? [Guido] That's already on the agenda. [Carol] For the short term we could use the docs community repo too.
+
+- I [Jim] appreciate how considerate this group is. The particular incident motivating my appreciation is that I got mentioned in the minutes of last meeting, and several people took pains to be sure that my name was presented there as I wanted it to be. Thank you.
+
+- [Ryan] We made the update to the TOML documentation we talked about on the last call and got my first Python pull request merged!
+
+## Discussion
+
+- [Petr] The `tkinter`, `pty`/`termios`, `sqlite3` (maybe more?) use `master/slave` terminology in API and docs. What's the best way to come up with, and switch to, better terms? Is there an expert I should talk to?
+  - [Carol] Thursday Bram has authored a Responsible Communications book which may have helpful guidelines. Other communities have used `leader/follower`. [Linux PR on the subject](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=49decddd39e5f6132ccd7d9fdc3d7c470b0061bb)
+  - [Hugo] Some suggestions: https://www.ietf.org/archive/id/draft-knodel-terminology-14.html#name-master-slave (Carol: I like the suggestions in this document for different usages.)
```

Comment:
Please turn this into a click able link.

Reply:
which part of the text exactly ?

Reply:
The URL:
`https://www.ietf.org/archive/id/draft-knodel-terminology-14.html#name-master-slave`

Reply:
Aren't links automatically clickable in MD?
See e.g. https://www.ietf.org/archive/id/draft-knodel-terminology-14.html#name-master-slave, with no special markup.

Reply:
This one wasn't clickable when I originally checked the preview.

Reply:
See also the hackmd link at the top.